php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -1146,7 +1146,7 @@ class NDB_BVL_Instrument extends NDB_Page
 
             //If the answer is empty (or its a group and one of answers in the
             //group is empty) then run the rules
-            if ($elvalue==="" || $flag==false) {
+            if ($elvalue==="" || $flag==false || is_null($elvalue)) {
```

Comment:
With a put request some of the values are `null` but do not get flagged by the check